### ISSUE TITLE
Dicas para criptografias futuras

### ISSUE THREAD

**Issue by pedroaf222** [renoirfaria/hardpath#70]
As criptografias que parecem ser becos sem saída podem também ter a ver com o filósofo italiano que ele crê ser a reencarnação, ele possuía teorias acerca da infinidade do universo, o círculo na parede pode ser uma espécie de mapa para localidades em uma infinidade teórica de mundos, pela discrição do artista plástico, ele crê que ele está completando o trabalho do filósofo, então os trabalhos do italiano podem dar uma ótima base para também entender as idéias de cada livro, facilitando a contextualização de cada obra manuscrita( Lembrem-se, ele ficou anos bolando isso, os resultados vão estar escondidos em todos os lugares, minhas referencias bibliográficas devem nos dar um norte de que caminho seguir, não será possível descobrir tudo antes de entender o que ele entendia)
1582 – O Candeeiro 1

583 – Sigillus sigillorum

1582 – De umbris idearum

1584 – A ceia das cinzas

1584 – Sobre a causa, princípio e uno

1584 – Acerca do infinito, do universo e dos mundos

1584 – O despacho da fera triunfante

1585 – Sobre os heróicos furores

1586 – A interpretação

1588 – Oração de Despedida

1591 – De minimo

1591 – Da magia

1591 – De monade

1591 – De immenso et innumerabilibus

As datas podem ser importantes e creio que o garoto tenha realmente conseguido terminar uma teoria incompreendida e incompleta, mas a decifração demorará

**Comment by TalitaMarques** [comment]
Então, primeiro temos que ter a mesma paciência que o Bruno teve quando fez tudo isso, só assim suas obras poderão ser decifradas!!!

**Comment by IndayaSpindola** [comment]
Sim, primeiro temos que decifrar os textos dele e  depois entender a mensagem que eles passam e so depois ter uma conclusao!
Agora nao adianta nada tentarmos ter conclusões sendo que os textos nao estao todos decifrados.
Vai ser trabalhoso mas vai valer a pena espero eu!

**Comment by Gjr77** [comment]
Vdd! Vou ter um pouco de cautela .....
